Hogajama/hogajama-domain/src/test/java/com/gepardec/hogarama/domain/sensor/SensorPropertiesTest.java

```diff
@@ -6,28 +6,27 @@
 
 import java.util.Optional;
 
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.runner.RunWith;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.Mock;
 import org.mockito.Mockito;
-import org.mockito.runners.MockitoJUnitRunner;
-
+import org.mockito.junit.jupiter.MockitoExtension;
 import com.gepardec.hogarama.domain.unitmanagement.cache.SensorCache;
 import com.gepardec.hogarama.testdata.TestSensors;
 
-@RunWith(MockitoJUnitRunner.class)
+@ExtendWith(MockitoExtension.class)
 public class SensorPropertiesTest {
 
     private static final String DEVICE_ID_NOTFOUND = "SomeSensor";
  
     @Mock
     private SensorCache sensorCache;
 
-    @Before
+    @BeforeEach
     public void setUpMethod() throws Exception {
-        Mockito.when(sensorCache.getByDeviceId(DEVICE_ID_NOTFOUND)).thenReturn(Optional.empty());
-        Mockito.when(sensorCache.getByDeviceId(DEVICE_GRUENER_GEPARD)).thenReturn(TestSensors.sensorGruenerGepard());
+        Mockito.lenient().when(sensorCache.getByDeviceId(DEVICE_ID_NOTFOUND)).thenReturn(Optional.empty());
```

Comment:
Lenient would not be necessary if we make two @Nested Test-Classes for each type
- GruenerGepard
- DeviceIdNotFound

Then the Mocks could be done for each class seperately.